installer/pkg/config-generator/tls.go

```diff
@@ -123,7 +132,8 @@ func (c *ConfigGenerator) GenerateTLSConfig(clusterDir string) error {
 	}
 
 	// Ingress certs
-	if copyFile(kubeCACertPath, ingressCACertPath); err != nil {
+	err = copyFile(filepath.Join(clusterDir, kubeCACertPath), filepath.Join(clusterDir, ingressCACertPath))
+	if err != nil {
 		return fmt.Errorf("failed to import kube CA cert into ingress-ca.crt: %v", err)
```

Comment:
Unrelated to this PR, but if we're going to bother setting an `ingressCACertPath` up at the top of this file, we might want to use `filepath.Base(ingressCACertPath)` here instead of hard-coding `ingress-ca.crt`.  We wouldn't want to update the variable definition and then forget to bump this.

That said, do we have a policy on when to use variables for the `generated/...` paths?  There's a preexising `ingressCACertPath`, but in this commit you're adding a hard-coded `generated/tls/etcd-client-ca.key` without a wrapping variable.  Personally, I prefer the hard-coded strings (how often will we change these?), but the precedent seems to be to use wrapping variables, so we might want variables for `generated/tls/etcd-client-ca.*` to be consistent.